chart/kube-flannel/tests/daemonset_test.yaml

```diff
@@ -0,0 +1,47 @@
+suite: test daemonset
+templates:
+  - daemonset.yaml
+tests:
+  - it: should have the correct api version
+    asserts:
+      - isKind:
+          of: DaemonSet
+      - isAPIVersion:
+          of: apps/v1
+      - equal:
+          path: metadata.name
+          value: kube-flannel-ds
+
+  - it: should have the correct image
+    set:
+      flannel.image.repository: cicker.io/flannel/flannel
+      flannel.image.tag: v555.44.3
+    asserts:
+      - equal:
+          path: spec.template.spec.containers[0].image
+          value: cicker.io/flannel/flannel:v555.44.3
```

Comment:
What is this repo?

Reply:
The not existing one, just to check the value is passed correctly and it's not passed from values or else. The same is the not existing tag.